lib/airtap.js

```diff
@@ -82,12 +63,19 @@ Zuul.prototype.browser = function (info) {
 
 Zuul.prototype.run = function (cb) {
   var self = this
-
   var config = self._config
 
-  self._setup(function (err, control_port) {
+  createControlServer(config, function (err, server) {
     if (err) return cb(err)
-    config.control_port = control_port
+
+    debug('control server active on port %d', server.port)
+    config.control_port = server.port
+
+    function exit (err, result) {
+      server.close(function (err2) {
+        cb(err || err2, result)
+      })
+    }
```

Comment:
Added this function to properly close the control server (and its `watchify` instance) before calling back with results. This is needed in order for `tape` to not hang.

Reply:
For a next PR: perhaps rename this function to `finish`, because `exit` is too much like `process.exit`.